test/test.bats

```diff
@@ -0,0 +1,23 @@
+setup() {
+    load 'test_helper/bats-support/load'
+    load 'test_helper/bats-assert/load'
+
+    DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd )"
+    export KEY=tata
```

Comment:
Can we use [the English version](https://fr.wikipedia.org/wiki/Variable_m%C3%A9tasyntaxique) of `toto, tata, tutu` : `foo, bar, baz`

Reply:
done